File: java/src/test/java/com/example/leetcode/TwoSumTest.java
package com.example.leetcode;

import org.junit.Test;

import static org.junit.Assert.*;

public class TwoSumTest {

    private TwoSum twoSum = new TwoSum();

    @Test
    public void test() {
        int[] nums = {2, 7, 11, 15};
        int target = 9;

        int[] result = twoSum.twoSum(nums, target);
        assertArrayEquals(new int[]{0, 1}, result);
    }
}

File: python/two_sum.test.py
import unittest
import two_sum

class TwoSumTest(unittest.TestCase):
  def test_1(self):
    nums = [2, 7, 11, 15]
    target = 9
    result = two_sum.Solution().twoSum(nums, target)
    self.assertEquals([0, 1], result)


unittest.main(verbosity=2)